src/ImagineSvg/Image.php

```diff
@@ -0,0 +1,461 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\ImagineSvg;
```

Comment:
I'd vote for putting all the image related classes into a general `Image` namespace.